api/data_explorer/__main__.py

```diff
@@ -146,6 +146,9 @@ def _get_facets():
 
     for facet_config in facets_config:
         field_name = facet_config['elasticsearch_field_name']
+        if not field_name in mapping['type']['properties']:
```

Comment:
Let's keep an eye on how sensitive this is, we might consider making this a warning instead of an error in the future.

Reply:
Why would this be a warning? Most likely they made a typo in `ui.json`.

Reply:
A warning would be more lenient regarding stuff like changing elasticsearch data, configs for multiple datasets, etc. I don't think we should make it a warning now, this was more of a comment than a suggestion.

Reply:
Yes, we can discuss in the future. But in general, I think error is better. For example, if Elasticsearch data changed and a field is no longer available, it's better to fail and let user know, so they can update to the right field.